lib/zendesk_apps_tools/server.rb

```diff
@@ -44,6 +43,18 @@ class Server < Sinatra::Base
       ZendeskAppsSupport::Installed.new([app_js], [installation]).compile_js
     end
 
+    get "/:file" do |file|
+      access_control_allow_origin
+      send_file File.join(settings.root, 'assets', file)
```

Comment:
At least I wasn't able to do `File.read File.join 'some_dir', '..', 'file_in_parent'`